src/EventLoop/Internal/AbstractDriver.php

```diff
@@ -558,8 +561,11 @@ private function createFiber(): void
     private function createCallbackFiber(): void
     {
         $this->callbackFiber = new \Fiber(function (): void {
+            /** @var \Fiber $fiber */
+            $fiber = \Fiber::getCurrent();
```

Comment:
I wonder if the circular reference to the fiber object is worth it here, it may cause destruction delays for non-resumed fibers. That's probably not worth the minor performance difference to save a call within `clear()`. Fine leaving the param on `clear()` though.

Reply:
If we don't need the param, I'd rather remove it and add it later if we really need that.

Reply:
Fine by me. I'm seeing no noticeable performance difference with or without passing the fiber instance, it's definitely within the margin of error.